executor/index_lookup_hash_join.go

```diff
@@ -45,6 +45,7 @@ type IndexNestedLoopHashJoin struct {
 	IndexLookUpJoin
 	resultCh          chan *indexHashJoinResult
 	joinChkResourceCh []chan *chunk.Chunk
+	joiners           []joiner
```

Comment:
Please add a comment here like [this](https://github.com/pingcap/tidb/blob/master/executor/join.go#L61-L62).